app/controllers/conferences_controller.rb

```diff
@@ -18,10 +18,14 @@ def create
     @conference.season_id = current_season.id
     @conference.gid = generate_gid(team)
 
-    if @conference.save
-      redirect_to edit_team_path(team), notice: 'Conference was successfully created.'
-    else
-      render action: :new
+    respond_to do |format|
+      if @conference.save
+        format.html { redirect_to edit_team_path(team), notice: 'Conference was successfully created.' }
+        format.js { render json: [@conference] }
```

Comment:
Why do you return an array instead of just one element? This being a `conferences/create` endpoint, I'd expect only a single element being returned:
```ruby
render json: @conference
```

Reply:
I know this is odd, but if I don't return an array, the ajax success function is not even called.

Reply:
Hm, looks more like a general format / parsing confusion here... 🤔 

Have you tried
```ruby
format.js { render json: @conference.to_json }
```

Plus: I'm not 100% sure if `remote: true` will automatically make the request a `format: script`one. It could also be a `JSON` request. Have you checked this?

Reply:
I tried .to_json but it didn't work. Also, remote:true looks for a js format. It doesn't respond to json.

Reply:
Sorry, didn't make myself clear in the previous comment 🙈 

You can set the content type yourself in the form `data: { type: :json }` and this may eventually fix your problem. For jQuery, `script` and `json` are two different things and it parses it differently. From what I remember for `script` / `js` it expects plain text, for `json` something different.

However, if you can fix it - yay - if not, we can also do it later ✌️

Reply:
You're right. The data-type=json fixed it.

Reply:
Ace 🎉  Just ping me again when you're done with all the changes.

Reply:
I'm done, @klappradla